mappings/048ee598172481f98d613fc9fb6a03fd6ec3edccb75f8e4afc993b6645584f.json

```diff
@@ -2,20 +2,20 @@
     "subject": "048ee598172481f98d613fc9fb6a03fd6ec3edccb75f8e4afc993b6645584f",
     "url": {
         "sequenceNumber": 0,
```

Comment:
Hi @Aljun4ik,

you should increment sequenceNumber for every updated value and sign json again.

Thanks!

Reply:
Done